VHM Content (#812)
SUSE/doc-susemanager: modules/client-configuration/pages/vhm-aws.adoc

[[vhm-aws]]
= VHM and Amazon Web Services

> You can use a {productname} VHM to manage Amazon Web Services (AWS) clusters.

Reviewer: ```suggestion
You can use a {productname} VHM to gather your instances from Amazon Web Services (AWS).
```

> The VHM allows {productname} to obtain and report information about your clusters, including the ability to do CVE audits, and report if containers are outdated.

Reviewer: Why are we mentioning containers here? In any case, using a VHM does not enable you to do CVE audits or getting info about patches. You must have those instances also registered as systems into SUSE Manager in order to manage them.

For more information on VHMs, see xref:client-configuration:vhm.adoc[].

== Create an Amazon EC2 VHM

The Virtual Host Manager (VHM) runs on the {productname} Server.

> To run a VHM, your {productname} Server will need to have port 443 open, in order to access the clients.

Reviewer: The `443` port is not a requirement here.
